### DISCUSSION TITLE
New UI Header Tabs

### DISCUSSION THREAD

**Discussion by CodingPupper3033** [inventree/InvenTree#6470] [Q&A] [answered]
Hello! I've been trying to wrap my head around the new ui's source code, and I can't figure something out. 

In Headers.tsx, function NavTabs, we have the variable tabValue to tell the tabs which one it's on. It uses parameters, so I looked at the router.tsx file, and I don't see any `:tabValue,` so how does the value get set?

Also, I think there's a minor bug based on this where if you use the browser bar to navigate to a page and not a tab, the header (and just the header) will think you're on the default home page. It's not a critical bug, but it exists.

**Comment by SchrodingersGat** [comment]
@CodingPupper3033 you might have found a bug here - it does appear that the `tabValue` variable is not really controlled. @matmair @wolflu05 any thoughts on this one?

**Reply by matmair** [reply]
Checking from my phone it seems like a repeatable bug

**Comment by XanderLuciano** [comment, marked as answer]
> I think there's a minor bug based on this where if you use the browser bar to navigate to a page and not a tab, the header (and just the header) will think you're on the default home page. 

This seemed like an ideal bug to get familiarized with this project. 😄 
Fixed in #6694
